apps/api-gateway/src/board/board.controller.ts

```diff
@@ -1,7 +1,55 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Post, Delete, Param, Body, Patch } from '@nestjs/common';
 import { BoardService } from './board.service';
+import { ApiCreatedResponse, ApiOperation } from '@nestjs/swagger';
+import { CreateBoardDto } from './dto/create-board.dto';
+import { UpdateBoardDto } from './dto/update-board.dto';
 
 @Controller('board')
 export class BoardController {
   public constructor(private readonly boardService: BoardService) {}
+
+  @Get()
+  @ApiOperation({ summary: '전체 게시글 조회' })
+  @ApiCreatedResponse({
+    description: '전체 게시글 조회',
+  })
+  public async getAllBoards() {
+    return await this.boardService.getAllBoards();
+  }
+
+  @Get(':id')
+  @ApiOperation({ summary: '게시글 상세 조회' })
+  @ApiCreatedResponse({
+    description: '게시글 상세 조회',
+  })
+  public async getOneBoard(@Param('id') postId: string) {
+    return await this.boardService.getOneBoard(postId);
+  }
+
+  @Post()
+  @ApiOperation({ summary: '게시글 생성' })
+  @ApiCreatedResponse({
+    description: '게시글 생성',
+  })
+  public async createBoard(@Body() createBoardDto: CreateBoardDto) {
+    return await this.boardService.createBoard(createBoardDto);
+  }
+
+  @Patch(':id')
+  @ApiOperation({ summary: '게시글 수정' })
+  @ApiCreatedResponse({
+    description: '게시글 수정',
+  })
+  public async updateBoard(@Param('id') updateId: string, @Body() updateData: UpdateBoardDto) {
```

Comment:
이렇게 표현해주는게 괜찮을 것 같습니다
```suggestion
  public async updateBoard(@Param('id') id: string, @Body() updateBoardDto: UpdateBoardDto) {
```

Reply:
해당 부분 수정 완료했습니다! 
수정된 커밋입니다 :  eaf4282ef0c842bc6e9759326bf7f477f9639f8f

Reply:
updateData도 updateBoardDto와 같이 dto라는 이름으로 표현되면 좋을 것 같습니다~

Reply:
앗 말씀해 주신 부분은 도중에 발견해서 여기서 커밋되었습니다😭
감사합니다! 
수정된 커밋입니다 : 07b4a48d690df0877b516528792074a9ef4cabfa